client/login/wp-login/hooks/get-header-text.tsx

```diff
@@ -24,11 +24,32 @@ interface Props {
 	isSocialFirst?: boolean;
 	isWooJPC?: boolean;
 	isWCCOM?: boolean;
+	isBlazePro?: boolean;
 	isFromAkismet?: boolean;
 	isFromAutomatticForAgenciesPlugin?: boolean;
 	isGravPoweredClient?: boolean;
+	isUserLoggedIn?: boolean;
 }
 
+const getLoggedInUserHeaderText = ( {
```

Comment:
Just thinking out loud - the Authorize screen might end up using the same call, and that also has a logged-in state. So we'd be rendering `Connect [OAuth2 client]`. But not sure yet - not sure if the same function would serve those headers/sub-headers too.

Nothing to do/change :-)

<img width="450" height="518" alt="Screenshot 2025-08-18 at 6 36 11 PM" src="https://github.com/user-attachments/assets/2810d795-b4a6-4fbf-ac9b-713d477eea3a" />